src/snowflake/snowpark/modin/plugin/extensions/dataframe_extensions.py

```diff
@@ -208,7 +208,7 @@ def to_pandas(
     **kwargs: Any,
 ) -> pandas.DataFrame:
     """
-    Convert Snowpark pandas DataFrame to `pandas.DataFrame <https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html>`_
+    Convert Snowpark pandas DataFrame to `pandas.DataFrame <https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html>`__
```

Comment:
should we remove the lines?

Reply:
I was previously running into a warning from Sphinx ("Duplicate explicit target name pandas.dataframe") without the double underscore, which turns it into an anonymous link. It seems to no longer be an issue so I'll revert it.